tests/entry-test.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var fs = require('fs');
+var expect = require('chai').expect;
+var Entry = require('../lib/entry');
+
+require('chai').config.truncateThreshold = 0;
```

Comment:
Funny, because I didn't know about this until looking at the other tests in this repo haha.